gooddata-sdk/gooddata_sdk/catalog/user/service.py

```diff
@@ -0,0 +1,97 @@
+# (C) 2022 GoodData Corporation
+from __future__ import annotations
+
+import functools
+from pathlib import Path
+from typing import List, Optional
+
+from gooddata_sdk.catalog.catalog_service_base import CatalogServiceBase
+from gooddata_sdk.catalog.user.declarative_model.user import CatalogDeclarativeUsers
+from gooddata_sdk.catalog.user.declarative_model.user_group import CatalogDeclarativeUserGroups
+from gooddata_sdk.catalog.user.entity_model.user import CatalogUser, CatalogUserDocument
+from gooddata_sdk.catalog.user.entity_model.user_group import CatalogUserGroup, CatalogUserGroupDocument
+from gooddata_sdk.utils import load_all_entities, load_all_entities_dict
+
+
+class CatalogUserService(CatalogServiceBase):
+    def list_users(self) -> List[CatalogUser]:
+        get_users = functools.partial(
+            self._entities_api.get_all_entities_users,
+            include=["userGroups"],
+            _check_return_type=False,
+        )
+        users = load_all_entities_dict(get_users, camel_case=False)
```

Comment:
Why is load_all_entities_dict needed in the case of users?

Reply:
It is because of the issue we talked before. In case of users the translation from camel case to snake case is not performed by client class. In case of user groups there are currently no keys that would be change because of conversion, so the old approach can be used.